calico_node/startup/startup_test.go

```diff
@@ -463,81 +698,82 @@ var _ = Describe("UT for Node IP assignment and conflict checking.", func() {
 	)
 })
 
```

Comment:
I believe the latest libcalico-go should support KDD for these types now.

Reply:
ok I pulled in the latest KDD and updated the tests